coverage/pom.xml

```diff
@@ -7,11 +7,11 @@
   <parent>
     <groupId>com.google.cloud.bigdataoss</groupId>
     <artifactId>bigdataoss-parent</artifactId>
-    <version>2.2.3-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
   </parent>
 
   <artifactId>coverage</artifactId>
-  <version>2.2.3-SNAPSHOT</version>
+  <version>3.0.0-SNAPSHOT</version>
```

Comment:
Are these versions related somehow? Can we introduce a maven property to keep them in sync?

Reply:
Maven does not like properties in artifact definition version field.